.github/workflows/backport.yml

```diff
@@ -50,14 +50,14 @@ jobs:
 
       - name: "Add 'has: port to stable' label"
         if: steps.backport.outputs.created_pull_numbers != ''
-        env:
-          # Not the app on purpose to avoid triggering another workflow run
-          # after adding this label
-          GH_TOKEN: ${{ github.token }}
-          REPOSITORY: ${{ github.repository }}
-          NUMBER: ${{ github.event.number }}
-        run: |
-          gh api \
-            --method POST \
-            /repos/"$REPOSITORY"/issues/"$NUMBER"/labels \
-            -f "labels[]=has: port to stable"
+        uses: actions/github-script@v7.0.1
+        with:
+          # Not using the app on purpose to avoid triggering another workflow
+          # run after adding this label.
+          script: |
+            await github.rest.issues.addLabels({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.payload.pull_request.number,
+              labels: [ '8.has: port to stable' ]
```

Comment:
The label seems to have had a different name previously:

https://github.com/nix-community/stylix/blob/1baa44cf8c3a4699d0beda91f39ba7942b46269d/.github/workflows/backport.yml#L63